utility/lambdas/ecs-asg-sizer/lambda_function.py

```diff
@@ -0,0 +1,115 @@
+import boto3
+import json
+import os
+
+session = boto3.session.Session()
+asgClient = session.client(service_name='autoscaling')
+ecsClient = session.client(service_name='ecs')
+
+#
+# Make an ASG the perfect size for an ECS cluster
+#
+def lambda_handler(event, context):
+    asgName = get_env('ASG_NAME')
+    clusterName = get_env('ECS_CLUSTER')
+    thresholdCpu = int(get_env('THRESHOLD_CPU'))
+    thresholdMemory = int(get_env('THRESHOLD_MEMORY'))
+    thresholdCount = int(get_env('THRESHOLD_COUNT'))
+
+    usages = get_usages(clusterName)
+    log_debug('Targeting threshold {0}x {1} cpu / {2} mem'.format(thresholdCount, thresholdCpu, thresholdMemory))
+    for usage in usages:
+        log_debug('    {0} - {1} / {2}'.format(usage['id'], usage['remainingCpu'], usage['remainingMemory']))
```

Comment:
We can do interpolation in py3.6 functions which make these less gross
https://cito.github.io/blog/f-strings/

Reply:
Just figured out i'm running 2.7 locally.  Whoops!